drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -157,53 +125,29 @@ static int i2c_parsing_return_value(const struct device *dev)
 	if (data->err == ETIMEDOUT)
 		return -ETIMEDOUT;
 
-	if (config->port < I2C_STANDARD_PORT_COUNT) {
-		/* The device does not respond ACK */
-		if (data->err == HOSTA_NACK)
-			return -ENXIO;
-		else
-			return -EIO;
-	} else {
-		/* The device does not respond ACK */
-		if (data->err == E_HOSTA_ACK)
-			return -ENXIO;
-		else
-			return -EIO;
-	}
+	/* The device does not respond ACK */
+	if (data->err == HOSTA_NACK)
```

Comment:
This function has a few missing enclosing brackets around if branch bodies. https://docs.zephyrproject.org/latest/contribute/index.html#coding-style

Reply:
Fixed. Thank you.